routes/productRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+
+const productController = require("../controllers/productController");
+
+router.get("/:categoryId", productController.productsInqury);
```

Comment:
- 엔드포인트는 /products로 받을 수 있게 해주세요.
- category_id는 path parameter가 아닌 query parameter로 받아주세요.
- query parameter로 받아야 필터링 조건이 category_id 뿐만 여러 개가 되었을 때도 처리가 가능합니다.
```suggestion
router.get("", productController.productsInqury);
```